### ISSUE TITLE
Error when try to use your component in Nextjs: Cannot find module 'tsup-tutorial/Button' or its corresponding type declarations.

### ISSUE THREAD

**Issue by Jared-Dahlke** [officialrajdeepsingh/tsup-tutorial#2]
i am trying to use the button component in your library.

it works fine with Vite, but with Nextjs I cannot build because i get this error:
`Cannot find module 'tsup-tutorial2/Button' or its corresponding type declarations.`

Any idea why this may be?

**Comment by officialrajdeepsingh** [comment]
You are not importing the package correctly. The correct ways.

```js
tsup-tutorial/Button
```

**Comment by Jared-Dahlke** [comment]
Nope. If you try to use I think next 13.2 or younger it won’t work because they have moduleResolution:node instead of bundled

**Comment by officialrajdeepsingh** [comment]
> Nope. If you try to use I think next 13.2 or younger it won’t work because they have moduleResolution:node instead of bundled

Yes, I've also found the same issue. Due to my busy schedule, I can not find the actual cause of the problem and also can't find a solution.

**Comment by officialrajdeepsingh** [comment]
> Nope. If you try to use I think next 13.2 or younger it won’t work because they have moduleResolution:node instead of bundled

Yes, I've also found the same issue. Due to my busy schedule, I can not find the actual cause of the problem and also can't find a solution.

**Comment by preshonyee** [comment]
@Jared-Dahlke I'm having the same error. What was the fix for this?

**Comment by Jared-Dahlke** [comment]
I found a solution.


for some reason you have to specify the exact path in package.json. So you have to have 2 exports per component. One for node and one for bundler 

![image](https://github.com/officialrajdeepsingh/tsup-tutorial/assets/32936978/1e8ee832-34b6-46ba-8d48-c32653249561)

And you have to import it from "/dist/button"

But again if you can just get your consumers to change to bundler in their config then you don't have to do any of this it just works
